src/Bridge/Symfony/Routing/IriConverter.php

```diff
@@ -92,7 +92,7 @@ public function getItemFromIri(string $iri, array $context = [])
     public function getIriFromItem($item, int $referenceType = UrlGeneratorInterface::ABS_PATH): string
     {
         $resourceClass = $this->getObjectClass($item);
-        $routeName = $this->routeNameResolver->getRouteName($resourceClass, false);
+        $routeName = $this->routeNameResolver->getRouteName($resourceClass, OperationType::ITEM);
```

Comment:
If we don't how would we now they're deprecated? ^^ Anyway, [there are a bunch of places where we still use `false`](https://github.com/api-platform/core/search?utf8=%E2%9C%93&q=getRouteName&type=), though we were going to fix those only when breaking.